modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition_manager.py

```diff
@@ -26,64 +25,7 @@
     PandasOnRayDataframeRowPartition,
 )
 from .partition import PandasOnRayDataframePartition
-from modin.core.execution.ray.generic.modin_aqp import call_progress_bar
 from modin.error_message import ErrorMessage
-import pandas
```

Comment:
This import is still necessary (it's used in `broadcast_apply`).

Reply:
This import is there, I just moved it to line 17. There is a certain order of imports that most python developers support. https://www.python.org/dev/peps/pep-0008/#imports
>Imports should be grouped in the following order:
>
>Standard library imports.
>Related third party imports.
>Local application/library specific imports."

I'm fixing the imports a bit where I see.